components/Main/Results.vue

```diff
@@ -0,0 +1,242 @@
+<script setup lang="ts">
+interface Stat {
```

Comment:
Let's create a main interface file to add these interfaces please and then import it here, maybe can be used later